components/ws-daemon/pkg/content/initializer.go

```diff
@@ -60,7 +67,7 @@ func collectRemoteContent(ctx context.Context, rs storage.DirectAccess, ps stora
 		}
 		info, err := ps.SignDownload(ctx, bkt, obj)
 		if err != nil {
-			return nil, xerrors.Errorf("cannot find snapshot: %w", err)
+			return nil, ErrCannotFindSnapshot
```

Comment:
@csweichel that's what you meant? 🤷🏻‍♂️

Reply:
Looks good. Only thing: we should check if the error actually is `storage.ErrNotFound`

Reply:
ok

Reply:
can there be any other error otherwise? so, should the other error check remain?

Reply:
Yes and Yes :)